packages/peer-exchange/src/waku_peer_exchange_discovery.ts

```diff
@@ -149,60 +149,53 @@ export class PeerExchangeDiscovery
     }, queryInterval * currentAttempt);
   };
 
-  private query(peerId: PeerId): Promise<void> {
-    return this.peerExchange.query(
-      {
-        numPeers: DEFAULT_PEER_EXCHANGE_REQUEST_NODES,
+  private async query(peerId: PeerId): Promise<void> {
+    const peerInfos = await this.peerExchange.query({
+      numPeers: DEFAULT_PEER_EXCHANGE_REQUEST_NODES,
+      peerId,
+    });
+
+    for (const _peerInfo of peerInfos) {
+      const { ENR } = _peerInfo;
+      if (!ENR) {
+        log("no ENR");
```

Comment:
We can see that this should not be possible from the code in [waku_peer_exchange.ts](https://github.com/waku-org/js-waku/pull/1193/files#diff-7d58f2441b91eee07006f585bcceb150b4621bb979fc527cd4b37506e1bd6824)

I don't think this log line is useful.

Reply:
Should be sorted with #1196